docker/celeryworker/Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /usr/src/app
 COPY ./backend/requirements.txt /usr/src/app/.
 
 RUN pip3 install -r requirements.txt
+RUN pip3 install watchdog argh
```

Comment:
Same comment, maybe put this before in the file.

Reply:
I don't know if we should add this to the requirements.txt as it's only a requirement for the dev settings (to be able to do the hot reload). Feel free to update.